_posts/2015-09-18-epages-you-2015.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Annual company event: ePagees rock the YO(U)"
```

Comment:
Please ask Chrissi what the official spelling for the YOU is. So far I've seen "YO(U)", "(YO)U", "YOU"...